packages/core/package.json

```diff
@@ -130,8 +130,10 @@
     "pg-transactional-tests": "^1.0.9",
     "supertest": "^7.0.0",
     "svelte": "^4.2.19",
+    "ts-morph": "^25.0.1",
+    "ts-node": "^10.9.2",
```

Comment:
why this? what does ts-morph. also we have svelte here. Do we need it?

Reply:
it's like recast used for large refactors

Reply:
How did you use in this case? Maybe a note in the README that we have this now at our disposal